pandas/plotting/_core.py

```diff
@@ -360,8 +360,7 @@ def _compute_plot_data(self):
 
         # no empty frames or series allowed
```

Comment:
Not a blocker for me but might be nice to clarify this comment. On initial glance it implies to me that the entire passed Frame might have been empty, whereas it really only y applies where numeric data doesn't exist, which is slightly different